src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -111,6 +111,17 @@
       - nfs-utils
   when: inventory_hostname == 'base-nfs'
 
+- name: Install packages for KDC base image
```

Comment:
Do we need to install the packages in CentOS images?

Reply:
CentOS include Fedora.yml, so it gets installed there.